cloudferrylib/utils/file_like_proxy.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 
+from progressbar import Bar, Percentage, ProgressBar
```

Comment:
It is recommended  to not importing objects, only modules.
http://docs.openstack.org/developer/hacking/#imports